src/app/Fake.Core.Target/Target.fs

```diff
@@ -50,6 +50,10 @@ and [<NoComparison>] [<NoEquality>] TargetContext =
     member x.HasError =
         x.PreviousTargets
         |> List.exists (fun t -> t.Error.IsSome)
+    member x.LastTargetError =
+        x.PreviousTargets
+        |> List.last
```

Comment:
strictly speaking LastTargetError needs to be a `bool option` as the `PreviousTargets` list might be empty...

Reply:
oh your so right...fixing now